.github/workflows/pull-request.yml

```
@@ -215,6 +204,12 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v6
 
+      - name: Setup Redis
+        uses: ./.github/actions/setup-redis
```

actionlint (1.7.12)

[error] description is required in metadata of "Setup Redis" action at "/home/jailuser/git/.github/actions/setup-redis/action.yml"

(action)